charts/external-dns/values.yaml

```diff
@@ -235,6 +235,9 @@ labelFilter:
 # -- Record types to manage (default: A, AAAA, CNAME)
 managedRecordTypes: []
 
+# -- Disable nodePort support (for environments where access to Node resources is forbidden)
+ignoreNodePorts: false
```

Comment:
Does this mean that
```
gnoreNodePorts: true
```

This should modify clusterroles and `recources["nodes"]`? here https://github.com/kubernetes-sigs/external-dns/blob/b41f3fdd90eae45c6cdfca8eb628563dd98ea2fd/charts/external-dns/templates/clusterrole.yaml#L9